### PR TITLE
Move GPU compute article from WebFu to web.dev

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR simply moves the ["GPU compute" article from Web Fu](https://developers.google.com/web/updates/2019/08/get-started-with-gpu-compute-on-the-web) to web.dev now that we have web.dev/gpu.
I'll start a PR to redirect automatically WebFu article to web.dev as well.

FYI @Kangz

Live preview: https://deploy-preview-6113--web-dev-staging.netlify.app/gpu-compute/

### CODE REVIEW

File: src/site/content/en/blog/gpu-compute/index.md [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+
+### GPU Buffers creation
+
+For the sake of simplicity, matrices will be represented as a list of floating
+point numbers. The first element is the number of rows, the second element the
+number of columns, and the rest is the actual numbers of the matrix.
+
+<figure class="w-figure">
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/IUv15DMl2yDwTGxeJNux.jpeg", alt="Simple representation of a matrix in JavaScript and it's equivalent in mathematical notation", width="800", height="158" %}
+  <figcaption>Simple representation of a matrix in JavaScript and it's equivalent in mathematical notation</figcaption>
```
Comment (samthor) [inline src/site/content/en/blog/gpu-compute/index.md:240]: grammar nit: its, not it's (it is)

File: src/site/content/en/blog/gpu-compute/index.md [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+
+The three GPU buffers are storage buffers as we need to store and retrieve data in
+the compute shader. This explains why the GPU buffer usage flags include
+`GPUBufferUsage.STORAGE` for all of them. The result matrix usage flag also has
+`GPUBufferUsage.COPY_SRC` because it will be copied to another buffer for
+reading once all GPU queue commands have all been executed.
+
+```js
+const adapter = await navigator.gpu.requestAdapter();
+if (!adapter) return;
```
Comment (samthor) [inline src/site/content/en/blog/gpu-compute/index.md:251]: {}'s around `return`

File: src/site/content/en/blog/gpu-compute/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/gpu-compute/index.md:240]: Fixed. Thanks!

File: src/site/content/en/blog/gpu-compute/index.md (hunk 2 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/gpu-compute/index.md:251]: Done.

File: src/site/content/en/blog/gpu-compute/index.md [hunk 3]
```diff
@@ -0,0 +1,612 @@
+---
+title: Get started with GPU Compute on the web 
+subhead: |
+  This article is about me playing with the experimental WebGPU API and sharing
+  my journey with web developers interested in performing data-parallel
+  computations using the GPU.
```
Comment (mihajlija) [inline src/site/content/en/blog/gpu-compute/index.md:6]:
```suggestion
  This post explores the experimental WebGPU API through examples
  and helps you get started with performing data-parallel
  computations using the GPU.
```

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/gpu-compute/index.md (hunk 3 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/gpu-compute/index.md:6]: Fixed. Thank you!

**Pull request merged**